pkg/lifecycle/versions.rules.go

```diff
@@ -146,5 +146,5 @@ func (vr *VersionRules) CheckChartVersionToRelease(chartVersion string) (bool, e
 		logrus.Errorf("failed to check version to release for chartVersion:%s with error:%v", chartVersion, err)
 		return false, err
 	}
-	return chartVersionInt == (vr.maxVersion - 1), nil
+	return chartVersionInt == (vr.MaxVersion - 1), nil
```

Comment:
The parenthesis aren't necessary.

Reply:
I agree it is not necessary. 

I intentionally wrote this way for easier reading. 
Can we keep this one?